src/pdf_parser/pdf_utils/parsing_utils.py

```diff
@@ -845,23 +885,33 @@ def _perform_ocr(
             except IndexError:
                 # No language was found in the GCV response
                 language = None
+        else:
+            raise ValueError(
+                "The OCR agent must be either a TesseractAgent or a GCVAgent."
+            )
+
+        # Heuristic to get rid of blocks with no text or text that is too short.
+        if len(text.split(" ")) < 3:
+            return None, None
```

Comment:
I feel like this should be a postprocessing step rather than hidden in a function?

Reply:
Moved to a function that can be improved and makes this more specific.